sdk/samples/iot/README.md

```diff
@@ -587,101 +572,39 @@ This section provides an overview of the different samples available to run and
+- *Executable:* `paho_iot_pnp_sample`
 
-  To interact with this sample, **you must use the Azure IoT Explorer**.
+  This [sample](./paho_iot_pnp_sample.c) connects an Azure IoT Plug and Play enabled device simulating a thermostat.  This device is described via the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). This sample demonstrates sending telemetry and properties from the device and receiving commands and writeable properties from the service.
 
-  <details><summary><i>How to interact with the Plug and Play sample:</i></summary>
-  <p>
+  X509 authentication is used to connect directly to Azure IoT Hub.
 
-    The capabilities are listed below.
+  The easiest way to interact with this sample from the service side is to use Azure IoT Explorer.  To use the sample:
 
-    <details><summary><b>Device Twin:</b></summary>
-    <p>
-
-    Two device twin properties are supported in this sample:
-    - A desired property named `targetTemperature` with a `double` value for the desired temperature.
-    - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature reached since device boot.
-    <br>
-
-    <b>To send a device twin desired property message:</b> Select your device's "Device Twin" tab in the Azure IoT Explorer. Add the property `targetTemperature` along with a corresponding value to the `desired` section of the device twin JSON. Select "Save" to update the document and send the twin message to the device.
-
-    ```json
-    "properties": {
-        "desired": {
-            "targetTemperature": 68.5,
-        }
-    }
-    ```
+  - Follow the initial setup instructions described above.
+  - Install [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
+  - Download [the thermostat model](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json) to a local directory.
+  - Build and run `paho_iot_pnp_sample`.
+  - Start Azure IoT Explorer and then:
+    - [Configure your hub](https://github.com/Azure/azure-iot-explorer/#configure-an-iot-hub-connection).  Once you've created your thermostat device, you should see it listed in the UX.
+    - Go to `IoT Plug and Play Settings` on the home screen, select `Local Folder` for the location of the model definitions, and point to the folder you downloaded the thermostat model.
+    - Go to the devices list and select your thermostat device.  Now select `IoT Plug and Play components` and then `Default Component`.
+    - You will now be able to interact with the Plug and Play device.
 
-    No other property names sent in a desired property message are supported. If any are sent, the log will report there is nothing to update.
+Additional instructions for Azure IoT Explorer, including screenshots, are available [here](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
+  
```

Comment:
To keep the description sections condensed (this readme is long), I highly recommend a drop down here.  Those who want to run the sample can open it up to see the info.  I'm guessing the sample has to run BEFORE being able to configure your hub -- based on your instructions it does look like a requirement.  Or does the order matter?   Also, why was the outline of how to interact with the sample taken out? That breaks from the readme approach of the other samples.
nit: trying to be consistent with how we format the tabs and locations to find things on Azure IoT Explorer or Azure Portal.  Other sample instructions use \" \", but if we want to go with \` `, then we should update all to that at some point.

```suggestion
<details><summary><i>How to interact with the Plug and Play sample:</i></summary>
<p>

First, configure your Azure IoT Explorer to recognize the Plug and Play model:

  - Download [the thermostat model](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json) to a local directory.
  - [Build and run](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/README.md#build-and-run-the-sample) `paho_iot_pnp_sample`.
  - Start Azure IoT Explorer:
    - [Configure your hub](https://github.com/Azure/azure-iot-explorer/#configure-an-iot-hub-connection).  Once you've created your thermostat device, you should see it listed in the UX.
    - Go to "IoT Plug and Play Settings" on the home screen, select "Local Folder" for the location of the model definitions.  Point to the folder in which you downloaded the thermostat model.
    - Go to the devices list and select your thermostat device.  Select "IoT Plug and Play components" and then "Default Component".
    - Additional instructions for Azure IoT Explorer, including screenshots, are available [here](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
 
You will now be able to interact with the Plug and Play device.
```

Then keep or update the existing instructions on specific interaction with the sample - Device Twin, Commands, Telemetry.

Reply:
This portion of the sample is in the `paho_iot_pnp_component_sample` part which is going to be fixed up in PR4b.  Sorry for any confusion here.  

RE interaction with the sample in Device Explorer UX, I don't want to be in that business in too great detail because if the UX changes our content will be stale.  I'll start a conversation with IoT Explorer team on this because I think they should document the nitty gritty for UX interaction so there's less likelihood for stale content as its same group who owns UX & docs.